### DISCUSSION TITLE
Date Range Picker component

### DISCUSSION THREAD

**Discussion by Lasoty** [microsoft/fluentui-blazor#1980] [Ideas]
One component that I really miss is Date Range Picker. It could look like this: 

![image](https://github.com/microsoft/fluentui-blazor/assets/23433922/a385b604-31bf-4d62-9bc3-ed2ec2c4c979)

After clicking on the picker, a dialog box would open to select the date range:

![image](https://github.com/microsoft/fluentui-blazor/assets/23433922/bba2bf9a-0497-454c-84d1-a32a4ef8087e)

As a reference point, I suggest using the MudBlazor control (https://mudblazor.com/components/daterangepicker), which in my opinion meets all expectations.

**Comment by dvoituron** [comment]
We already discussed about that, including here #1893.

It is not currently present in FluentUI components and we have no plans to develop it.
It is, however, very simple to allow the user to select a range of dates, using two **FluentDatePicker** components.

We are always open to new PR :-)
